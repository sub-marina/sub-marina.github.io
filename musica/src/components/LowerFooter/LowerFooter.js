import React from 'react';

const LowerFooter = () => {
    return (
        <div className='lower-footer'>
            <div className='container'>
                <nav className='lower-footer__nav lower-nav'>
                    <ul>
                        <li><a href='#' className='lower-nav__link'>Home</a></li>
                        <li><a href='#' className='lower-nav__link'>Portfolio</a></li>
                        <li><a href='#' className='lower-nav__link'>Sitemap</a></li>
                        <li><a href='#' className='lower-nav__link'>Contact</a></li>
                    </ul>
                </nav>
                <p className='lower-footer__copyright'>Musica &#64;2013 by PremiumCoding | All Rights Reserved</p>
            </div>
        </div>
    );
}

export default LowerFooter;