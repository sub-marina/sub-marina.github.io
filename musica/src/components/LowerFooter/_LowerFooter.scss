.lower-footer {
  height: 35px;
  border-top: 1px solid #000000;
  background-color: #1b1b1b;
  font-size: 11px;
  text-shadow: 0 1px 0 rgba(0, 0, 0, 0.5);
  letter-spacing: -0.11px;
  color: #6e6e6e;
  line-height: 35px;

  .lower-nav ul {
    float: left;
    width: 192px;
    @include flex(space-between);
  }

  &__copyright {
    float: right;
  }
}