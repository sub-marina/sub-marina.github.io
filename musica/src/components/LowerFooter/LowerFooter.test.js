import React from 'react';
import { render } from '@testing-library/react';
import LowerFooter from './LowerFooter';

describe('Testing Footer.js', () => {
    test('Smoke test', () => {
        const { container } = render(<LowerFooter />);
        console.log(container.innerHTML);
    });

    test('Footer snapshot', () => {
        const { container } = render(<LowerFooter />);
        expect(container.innerHTML).toMatchSnapshot();
    });
});