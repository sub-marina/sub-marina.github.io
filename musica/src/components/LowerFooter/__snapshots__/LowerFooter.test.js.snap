// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Footer.js Footer snapshot 1`] = `"<div class=\\"lower-footer\\"><div class=\\"container\\"><nav class=\\"lower-footer__nav lower-nav\\"><ul><li><a href=\\"#\\" class=\\"lower-nav__link\\">Home</a></li><li><a href=\\"#\\" class=\\"lower-nav__link\\">Portfolio</a></li><li><a href=\\"#\\" class=\\"lower-nav__link\\">Sitemap</a></li><li><a href=\\"#\\" class=\\"lower-nav__link\\">Contact</a></li></ul></nav><p class=\\"lower-footer__copyright\\">Musica @2013 by PremiumCoding | All Rights Reserved</p></div></div>"`;
